setup.cfg

```
@@ -28,7 +28,7 @@ python_requires = >=3.10
 install_requires =
     # core packages
     scikit_image>=0.14.2
-    dipy>=1.8.0,<1.9.0
+    dipy>=1.10.0,<1.11.0
```

Do we really need the `1.11.0` here? The newly-released version is 1.10.0
```suggestion
    dipy>=1.10.0
```

Otherwise, why not `dipy==1.10.0`?

---

This is what we have done in the past, so that when a new dipy is released, pyAFQ doesn't immediately break (we specifically update pyAFQ whenever a new DIPY version is released)